settings/trex.toml

```diff
@@ -13,6 +13,9 @@ predefined = "web_mercator"
 
 [[tileset]]
 name = "vapour_trail"
+attribution = "<a href='http://junglebus.io/'>Jungle Bus</a> <a href='http://www.openstreetmap.org/copyright'>© OpenStreetMap contributors</a>"
+#center = 0, 0 # longitude, latitude
+start_zoom = 10
```

Comment:
what does this param do ? I can't find it in the [tilejson spec](https://github.com/mapbox/tilejson-spec/tree/master/2.2.0) ?

Reply:
It is the zoom part of center.